adafruit_dotstar.py

```diff
@@ -157,7 +164,7 @@ def _set_item(self, index, value):
         if isinstance(value, int):
             rgb = (value >> 16, (value >> 8) & 0xff, value & 0xff)
 
-        if len(value) == 4:
+        if len(rgb) == 4:
```

Comment:
If `value` was `int` this fails with:
`TypeError: object of type 'int' has no len()`
When using `int` color value instead of `tuple`.
Looking at the code it seemed clear this was supposed to be `rgb` all along as this would never work with an `int` value.